pyfr/integrators/dual/pseudo/multip.py

```diff
@@ -13,7 +13,7 @@
 
 class DualMultiPIntegrator(BaseDualPseudoIntegrator):
     def __init__(self, backend, systemcls, rallocs, mesh, initsoln, cfg,
-                 tcoeffs):
+                 tcoeffs, dt):
```

Comment:
Am I correct in thinking that we never use this parameter?

Reply:
Need to pass it onwards on line 111, to construct the pseudointegrators for each level
self.pintgs[l] = lpsint(backend, systemcls, rallocs, mesh,
                                    initsoln, cfg, tcoeffs, dt)